fastlane/Fastfile

```diff
@@ -72,7 +72,7 @@ ENV["PUBLIC_CONFIG_FILE"]=File.join(PROJECT_ROOT_FOLDER, "config", "Version.Publ
 ENV["INTERNAL_CONFIG_FILE"]=File.join(PROJECT_ROOT_FOLDER, "config", "Version.internal.xcconfig")
 ENV["DOWNLOAD_METADATA"]="./fastlane/download_metadata.swift"
 ENV["PROJECT_ROOT_FOLDER"]=PROJECT_ROOT_FOLDER + "/"
-
+ENV["APP_STORE_STRINGS_FILE_NAME"]="AppStoreStrings.po"
```

Comment:
🤔 I'm surprised that this is injectable, I thought the release toolkit hard-coded that 🤷 

Why did that env var suddenly started to be needed (or, if it was always needed, how did it manage to work until now then?)

Reply:
@jkmassel I'm curious about this, too.

According to GitHub, there's only [one occurrence of `APP_STORE_STRINGS_FILE_NAME` in the release toolkit](https://github.com/wordpress-mobile/release-toolkit/search?q=APP_STORE_STRINGS_FILE_NAME), and it has no fallback if the value is not set.

I had a quick look through the blame history, and I couldn't see a fallback being there in recent the past either.

How has this been working and not crashing when Ruby couldn't find this value 🤔 🤔 🤔